.drone.yml

```diff
@@ -3,7 +3,36 @@ type: docker
 name: root
 
 steps:
+  - name: check-sdk-abi
+    image: node:18.19
+    commands:
+      - cd packages/sdk
+      - npm ci
+      - npm run codegen
+      - npm run check-types
```

Comment:
Really necessary here?
`npm run codegen` yes definitely that's the goal of the PR, but `npm run check-types`, why?